steamapi/user.py

```diff
@@ -140,6 +143,16 @@ def __hash__(self):
         return hash(('user', self.id))
 
     # PRIVATE UTILITIES
+    @classmethod
```

Comment:
You don't need any methods or attributes of `SteamUser` (i.e.: `cls` is unused), change this to `@staticmethod`.